db.js

```diff
@@ -10,3 +10,38 @@ export function dbTrips() {
     },
   ];
 }
+
+export function dummyDestinations() {
+  return [
+    {
+      id: 1,
```

Comment:
Your ids should be strings. If they are numbers it will likely cause problems elsewhere comparing them to other values.

Reply:
Done